.github/workflows/release-snap.yaml

```diff
@@ -43,23 +53,15 @@ jobs:
       - name: yq - portable yaml processor
         uses: mikefarah/yq@v4
 
+      - if: ${{ matrix.confinement == 'classic' }}
+        name: Convert to Classic Snap
+        run: ./make-classic.sh
```

Comment:
Are we completely sure that we build the `strict` first and then the `classic`?


Because if for some reason the `classic` build runs first, then we are going to build the `strict` with a modified snapcraft.yaml

I suspect that matrix follows this order: `[ "strict", "classic" ]`... but just in case I would add an `else`

Reply:
Each job is isolated!
![image](https://github.com/user-attachments/assets/b5fa0283-9337-4de4-9d8a-941bf050ddc4)